specs.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+# Specifications
+
+- [Design documents](https://github.com/WebAssembly/design): documents describing the design, goals and high-level overview of WebAssembly.
```

Comment:
```suggestion
- [Original design documents](https://github.com/WebAssembly/design): documents describing the design, goals and high-level overview of WebAssembly. Same of these documents are outdated by now.
```

Reply:
Interesting; are they not relevant anymore for new additions?

Reply:
Note that I've already updated things like "Binary encoding", "Text format" and "Semantics" to redirect to the up-to-date spec and was thinking about removing them from the upstream `design` repo to avoid confusion if possible.

However, I thought that most other docs still apply to WebAssembly today - if not, maybe we should work out which ones should be split out in their own folder or repo as "legacy" / "original".

Reply:
Most of the design docs haven't been updated in a long time. I can't tell off-hand what's still in sync, but expect there to be small deviations throughout. More importantly, these docs do not cover any of the new features, whose design docs are in the respective proposal repos.

FWIW, I don't think we should delete any of the original design docs. They are valuable as a documentation for the rationale behind the original MVP design. But we need to make clear that they are no longer an up-to-date reflection of the status quo.

Reply:
> FWIW, I don't think we should delete any of the original design docs. They are valuable as a documentation for the rationale behind the original MVP design. But we need to make clear that they are no longer an up-to-date reflection of the status quo.

Agreed, which is why I think that moving them into own folder (or a repo) might be the best of both worlds.

Reply:
I've raised another issue on the upstream repo for that. I think this change is fine, but when we separate up-to-date documents from historical ones, we should revise this item again.

Reply:
Merged & moved them to the end of the list, since they sound even more niche for most users than initially thought.